lib/generators/sufia/install_generator.rb

```diff
@@ -172,8 +172,8 @@ def inject_routes
         "  # This behavior seems to show up only in production mode.\n" \
         "  mount Sufia::Engine => '/'\n"
 
-      sentinel = /devise_for :users/
-      inject_into_file 'config/routes.rb', routing_code, after: sentinel, verbose: false
+      sentinel = /end\Z/
```

Comment:
@jcoyne How does this ensure the Sufia engine is mounted on the last line? Won't this match the first `end` statement found? Or does `/end\Z/` match the first `end` statement with no whitespace on either side of it (which, presumably, will be on the last line of the file?